src/common/snapshots.rs

```diff
@@ -128,8 +128,13 @@ pub async fn recover_shard_snapshot(
                     let client = client.client(api_key.as_deref())?;
 
                     let (snapshot_path, snapshot_temp_path) =
-                        snapshots::download::download_snapshot(&client, url, download_dir.path())
-                            .await?;
+                        snapshots::download::download_snapshot(
+                            &client,
+                            url,
+                            download_dir.path(),
+                            false,
```

Comment:
We explicitly only allow `http://` and `https://` URLs for shard snapshots (and local files are handled explicitly, without relying on `file://` URLs), so I'd enforce `only_snapshots_dir` here, just in case.

```suggestion
                            true,
```

Reply:
Sure. Lets wait with this one until we've fixed actually downloading to the correct directory.